doc/advanced_usage.md

```diff
@@ -0,0 +1,88 @@
+# Advanced usage
+
+## Posting color-magnitude data
+
+For stellar targets we are interested in showing the HR diagram
+based on the color-magnitude data from a cross match to, e.g., the Gaia catalog.
+This is done by posting an `Annotation` to the source, with correctly formatted data.
+
+First lets see how to post an `Annotation` using the `request` package.
+Here is a working example:
+
+```
+import requests
+import json
+
+url = http://localhost:5000
+token = '239868fa-8307-41ad-983f-4a8180609df6'
+header = {"Authorization": f"token {token}", "content_type": "application/json"}
+parameters = {}
+data = {'obj_id': '2021example',
+        'origin': 'cross_match_robot',
+        'Gaia': {'Mag_G', 10.2, 'Mag_Bp': 9.8, 'Mag_Rp': 10.5, 'Plx': 8.5}
+        }
+
+r = requests.post(
+    f'{url}/api/annotation',
+    header=header,
+    params=parameters,
+    data=json.dumps(data))
+
+```
+
+Lets look at this one by one.
+
+The `url` should point to the SkyPortal instance,
+in this case running on a local machine.
+
+The `token` should be generated for your user
+through the profile page.
+The token must have the ACL to annotate.
+
+The `header` uses the token above
+and defines what kind of HTTP call is expected.
+
+The `parameters` are used to define query parameters,
+which are often very useful in GET calls,
+but in this case are left empty.
+Query parameters can also be appended to the URL
+using a `?` before each `keyword=value` pair,
+and each pair is separated using a `&`.
```

Comment:
I put this in because this is supposed to be an example for how to use the `request` module to interact with the API. 
For completeness I left this in. 
I think if a user is impatient and only looks at this example, they'll be happy to see all the most commonly used options. 
Unless you think there is going to be another example for a `GET` documented nearby where `params` will be used.